spec/factories/user.rb

```diff
@@ -5,6 +5,8 @@
     display_name "jgu"
     password "gu"
     role "default"
+    provider "twitter"
+    uid "abc123"
```

Comment:
I agree with your point but I don't think it's a good use of resources and time as it will break tests I believe.